### ISSUE TITLE
Method not found containsKey on Map

### ISSUE THREAD

**Issue by cuifengcn** [ethanblake4/dart_eval#147]
```dart
final testMap = {'default': ''};
print(testMap.containsKey('default'));
```
When starting compile, it has this error
```
flutter: │ ⛔ CompileError: Method not found containsKey on Map at "testMap.containsKey(..." (file package:example/main.dart)
flutter: │ ⛔ #0      Compiler.compileSources (package:dart_eval/src/eval/compiler/compiler.dart:497:7)
flutter: │ ⛔ #1      Compiler.compile (package:dart_eval/src/eval/compiler/compiler.dart:164:12)
```

dart_eval 0.7.1

**Comment by DeleMike** [comment]
Hi @ethanblake4 and @cuifengcn,

I would love to work on this issue. Can you elaborate more on what it entails?

**Comment by ethanblake4** [comment]
@DeleMike Sure, this is a great place to start on dart_eval as it just involves making standard library bindings. I'd recommend reviewing [this PR](https://github.com/ethanblake4/dart_eval/pull/112) to see an example of some functions that were added to List and Iterable. In this case you'd be adding the function to $Map in [map.dart](https://github.com/ethanblake4/dart_eval/blob/master/lib/src/eval/shared/stdlib/core/map.dart).

**Comment by ethanblake4** [comment]
Released in v0.7.2
